src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs

```diff
@@ -10,6 +10,10 @@ public abstract class CommonToastOptions
         /// </summary>
         public string ToastClass { get; set; } = Defaults.Classes.Toast;
 
+        /// <summary>
+        /// This variable applies to the title and message. If false HTML will be implemented, else HTML markup will be shown. />.
+        /// </summary>
+        public bool EscapeHtml { get; set; } = false;
```

Comment:
I think that, for a security concern, the text should be escaped by default: what do you think about it?

```suggestion
        public bool EscapeHtml { get; set; } = true;
```

Reply:
Not sure what happened the first time I commented, but it seemed to go away. 
My comment is this: I followed the same convention as Toastr.js.